src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
+    }
+
+    @Override
+    public void stopWork() {
+        return "certain machines stopped working";
```

Comment:
The message 'certain machines stopped working' is not informative. It should clearly state the type of machine, such as 'Excavator stopped working'. Additionally, ensure that the messages are capitalized properly for consistency and readability.